src/platform/brand-consolidation/containers/VetsDotGov.jsx

```diff
@@ -10,7 +10,7 @@ function VetsDotGov({ vetsDotGovEnabled, children }) {
 
 export function mapStateToProps(state) {
   return {
-    vetsDotGovEnabled: !state.buildSettings.brandConsolidationEnabled
+    vetsDotGovEnabled: !state.buildSettings.brandConsolidation.enabled
```

Comment:
Might be worth making a standalone selector for getting this enabled flag, since you're checking it in a couple places.